resources/public/api/conf/3.0/common/errors.yaml

```diff
@@ -0,0 +1,236 @@
+        it is after the maximum tax year value.
+      value:
+        code: RULE_TAX_YEAR_NOT_SUPPORTED
+        message: The tax year specified does not lie within the supported range
+
+    ruleTaxYearRangeInvalid:
+      description: Tax year range invalid. A tax year range of one year is required
+      value:
+        code: RULE_TAX_YEAR_RANGE_INVALID
+        message: Tax year range invalid. A tax year range of one year is required
+
+    formatCalculationId:
+      description: The format of the supplied calculation ID is invalid.
+      value:
+        code: FORMAT_CALCULATION_ID
+        message: The provided calculation ID is invalid
+
+    ruleTypeOfBusinessIncorrectSE:
+      description: The calculation ID supplied does not relate to a self-employment business.
+      value:
+        code: RULE_TYPE_OF_BUSINESS_INCORRECT
+        message: The calculation ID supplied relates to a different type of business
+
+    ruleTypeOfBusinessIncorrectForeign:
+      description: The calculation ID supplied does not relate to a foreign property business.
+      value:
+        code: RULE_TYPE_OF_BUSINESS_INCORRECT
+        message: The calculation ID supplied relates to a different type of business
+
+    ruleTypeOfBusinessIncorrectUk:
+      description: The calculation ID supplied does not relate to a UK property business.
+      value:
+        code: RULE_TYPE_OF_BUSINESS_INCORRECT
+        message: The calculation ID supplied relates to a different type of business
+
+    ruleSummaryStatusInvalid:
+      description: The status of the summary with this calculation ID is invalid.
+      value:
+        code: RULE_SUMMARY_STATUS_INVALID
+        message: Periodic data has changed. Request a new summary
+
+    ruleSummaryStatusSuperseded:
+      description: A newer summary calculation exists for this accounting period.
+      value:
+        code: RULE_SUMMARY_STATUS_SUPERSEDED
+        message: A newer summary calculation exists for this accounting period
+
+    rulePeriodicDataIncomplete:
+      description: One or more periodic updates are missing for this accounting period.
+      value:
+        code: RULE_PERIODIC_DATA_INCOMPLETE
+        message: One or more periodic updates are missing for this accounting period
+
+    ruleEndDateBeforeStartDate:
+      description: The accounting period end date value is before the start date.
+      value:
+        code: RULE_END_DATE_BEFORE_START_DATE
+        message: The supplied accounting period end date is before the start date
+
+    ruleAccountingPeriodNotSupported:
+      description: |
+        The specified accounting period is not supported, that is, the accounting period specified falls before the 
+        minimum tax year value.
+      value:
+        code: RULE_ACCOUNTING_PERIOD_NOT_SUPPORTED
+        message: |
+          The specified accounting period is not supported, that is, the accounting period specified falls before the
```

Comment:
In the code the error message for this is `The accounting period is not supported, because it predates the earliest allowable tax year`